services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -80,7 +80,7 @@ data class Hanke(
     //
     @JsonView(ChangeLogView::class)
     @field:Schema(description = "Hanke founder contact information")
-    var perustaja: Perustaja? = null,
+    var perustaja: Perustaja,
```

Comment:
I think we can't merge this until https://helsinkisolutionoffice.atlassian.net/browse/HAI-1348 has been done in frontend. Otherwise, we'll break hanke creation from the UI, if I'm not mistaken?

Reply:
Perustaja field is nullable again, but yes: this cannot be merged before the frontend is implemented. Creating a hanke without Perustaja will fail validation.